pkg/auth/authenticator/password/htpasswd/htpasswd.go

```diff
@@ -194,10 +197,11 @@ func testMD5Password(password, hash string) (bool, error) {
 }
 
 func testCryptPassword(password, hash string) (bool, error) {
-	// if len(password) > 8 {
-	// 	password = password[:8]
-	// }
-	// salt := hash[0:2]
-	// hash = hash[2:]
-	return false, errors.New("crypt password hashes are not supported")
+	testhash, err := util.Crypt(password, hash)
```

Comment:
are you saying that crypt() ignores the content of salt past the first two characters (for DES) or past the third $ (for MD5)? I didn't get that from the docs, but maybe I'm misreading them